src/guides/wolfpack/shapeshifter.mdx

```diff
@@ -13,14 +13,14 @@ The Shapeshifter is a powerful werewolf, able to turn the tide of an otherwise h
 - The Shapeshifter, both before and after he shifts, is seen as a member of the Wolfpack by the Seer.
 - The Shapeshifter is seen as a user of witchcraft.
 - The Shapeshifter is seen visiting by the Harlot/Stalker/Familiar when they are the killing wolf.
-- The Shapeshifter's secondary ability - activating the identity swap of their shift is classed as a self visit.
+- The Shapeshifter's secondary ability - activating the identity swap of their shift is classed as a self visit but is only seen by the Harlot.
```

Comment:
All self visits are only seen by the Harlot (self-prots, Messiah sacrifices, etc.). I think this and the other line should be worded in a way that doesn't imply that this is a specific exception to self-visits the Stalker can otherwise see. Perhaps the line before this could read "If the Shapeshifter kills, they are seen visiting by the Harlot/Stalker/Familiar" and this line could read "If the Shapeshifter transforms, they are seen visiting themselves by the Harlot". Or maybe this line could say "The Shapeshifter's secondary ability - activating the identity swap of their shift is classed as a self visit. Like other self-visits, this is only seen by the Harlot."

Reply:
Yeah, I'd go with what Hans suggests, with a minor tweak - simplify both of these and remove the mention of other roles here so it's just using standard terms like primary and secondary visits, then give this extra info in the Notes section. This makes this section easier to skim and removes the duplication.

Reply:
What if you replaced "but" with "and therefore" or "and therefore (like any other kind of self-visit, such as self-prots, Messiah sacrifices, etc.)"